trueline.sh

```diff
@@ -407,6 +425,7 @@ if [[ "${#TRUELINE_SEGMENTS[@]}" -eq 0 ]]; then
     declare -a TRUELINE_SEGMENTS=(
         'user,black,white,bold'
         'venv,black,purple,bold'
+        # 'conda_env,black,soft_red,bold'
```

Comment:
We don't have a `soft_red` color in the default colors. Change it to `purple` to be consistent with the standard python venv.